advanced-usage/index.md

```diff
@@ -86,19 +92,23 @@ to connect a different control computer to a vehicle and load the familiar contr
 
 #### Default Profiles
 
-Cockpit includes default profiles for submarine and boat use-cases, which cannot be edited. These serve as
-a reference for a recommended base profile, and are useful for restoring to a known reasonable interface in
-case something goes wrong with custom interface options. The default profiles are not persistent, so they
-may change through different Cockpit versions.
+Cockpit includes internal default profiles for submarine and boat use-cases. These serve as a reference 
+for a recommended base profile, and are useful for restoring to a known reasonable interface in case
+something goes wrong with custom interface options. The default profiles are not persistent, so they
+may change between different Cockpit versions.
```

Comment:
"Those are the profiles that come with Cockpit. By default they are copied for you as editable user profiles" or something like that. You can write it better definetly 😅